proxy/wireguard/tun_linux.go

```diff
@@ -199,6 +199,8 @@ func createKernelTun(localAddresses []netip.Addr, mtu int, handler promiscuousMo
 
 		r := netlink.NewRule()
 		r.Table, r.Family, r.Src = ipv6TableIndex, unix.AF_INET6, addr.IPNet
```

Comment:
what happens to this rule? I think it is simply overwritten

Reply:
I forget to `append()`, my fault.

Reply:
But my local repository didn't, so maybe yesterday I was too sleepy so I forgot to Ctrl + S 🥲